src/Symfony/Bridge/Doctrine/Tests/Validator/DoctrineLoaderTest.php

```diff
@@ -175,9 +187,33 @@ public function testClassValidator(bool $expected, string $classValidatorRegexp
     public function regexpProvider()
     {
         return [
-            [false, null],
-            [true, '{^'.preg_quote(DoctrineLoaderEntity::class).'$|^'.preg_quote(Entity::class).'$}'],
-            [false, '{^'.preg_quote(Entity::class).'$}'],
+            [true, null],
+            [true, '{^'.preg_quote(DoctrineLoaderEntity::class, '{').'$|^'.preg_quote(Entity::class, '{').'$}'],
+            [false, '{^'.preg_quote(Entity::class, '{').'$}'],
```

Comment:
the 2nd args to preg_quote are not needed, let's remove them to reduce the diff

Reply:
If removed, PHPStorm (or PHP Inspections, I'm not sure) display a big error...

Reply:
We don't care, tools don't have a brain :)